Add help files for new variation related cmdlets (#385)
MicrosoftDocs/OfficeDocs-SharePoint-PowerShell: Repair-SPVariation.md

---
external help file: Microsoft.SharePoint.Publishing.dll-Help.xml
Module Name: SharePointServer
online version:
schema: 2.0.0
---

# Repair-SPVariation

## SYNOPSIS
Corrects invalid variations system data that is found.

## SYNTAX

```
Repair-SPVariation -Identity <SPWebPipeBind> [-Recurse] [-Label <String>]
 [-AssignmentCollection <SPAssignmentCollection>] [-WhatIf] [-Confirm] [<CommonParameters>]
```

## DESCRIPTION
The command corrects invalid variations system data that is found.
If the recurse parameter is used, fixes are done recursively for all subsites.

## EXAMPLES

### EXAMPLE
```
C:\PS> Repair-SPVariation -Identity https://server/sites/pub/vhome/source/newsub -Recurse
```

> The example is to correct variations system data for the source variation on site https://server/sites/pub/vhome/source/newsub and all its subsites.

Reviewer: ```suggestion
This example corrects variations system data for the source variation on site https://server/sites/pub/vhome/source/newsub and all its subsites.

```

Parameter Sets: (All)
Aliases:

Required: True
Position: Named
Default value: None
Accept pipeline input: True (ByValue)
Accept wildcard characters: False
```

### -Label
Name of the label of the variation target.

```yaml
Type: String
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -Recurse
Fixes all subsites of the site specified by the Identity parameter.

```yaml
Type: SwitchParameter
Parameter Sets: (All)
Aliases:

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -Confirm
Prompts you for confirmation before running the cmdlet.

```yaml
Type: SwitchParameter
Parameter Sets: (All)
Aliases: cf

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -WhatIf
Shows what would happen if the cmdlet runs.
The cmdlet is not run.

```yaml
Type: SwitchParameter
Parameter Sets: (All)
Aliases: wi

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### CommonParameters

> This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](http://go.microsoft.com/fwlink/?LinkID=113216).

Reviewer: ```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).

```
